wiki/Guides/Beginner's_Tutorial/tr.md

```diff
@@ -0,0 +1,152 @@
+- [fripSide - a little christmas time (Darksonic) \[Easy\]](https://osu.ppy.sh/beatmaps/191008)(1.03 Yıldız)
+- [Yoko Shimomura - A Twinkle in the Sky (MoodyRPG) \[Easy\]](https://osu.ppy.sh/beatmaps/220650)(1.05 Yıldız)
+- [Sean Paul - She Doesn't Mind (Daniel Ngo Remix) (BounceBabe) \[Easy\]](https://osu.ppy.sh/beatmaps/158917)(1.08 Yıldız)
+- [Anamanaguchi - This is the End (Kurai) \[Average Joe\]](https://osu.ppy.sh/beatmaps/74686)(1.09 Yıldız)
+- [Infected Mushroom - The Pretender (Nemis) \[Easy\]](https://osu.ppy.sh/beatmaps/240979)(1.10 Yıldız)
+- [Do As Infinity - Fukai Mori (Andrea) \[Easy Dan\]](https://osu.ppy.sh/beatmaps/49478)(1.11 Yıldız)
+- [Nishiura Tomohito - Shop (samplefish) \[Easy\]](https://osu.ppy.sh/beatmaps/404645)(1.13 Yıldız)
+- [Angela Chang - Aurora (Pasha_Khvan97) \[Easy\]](https://osu.ppy.sh/beatmaps/397975)(1.14 Yıldız)
+
+### Acemi
+
+Biraz oynadıktan sonra, şu beatmapleri deneyin.
+
+- [Mutsuhiko Izumi - Russian Snowy Dance (Chocopikel) \[Easy\]](https://osu.ppy.sh/beatmaps/124348)(1.21 Yıldız)
+- [goreshit - o'er the flood (grumd) \[Easy\]](https://osu.ppy.sh/beatmaps/177139)(1.22 Yıldız)
+- [Hatsune Miku - Electric Love (Short Ver.) (KayeSara) \[Easy\]](https://osu.ppy.sh/beatmaps/141749)(1.25 Yıldız)
+- [DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE (Mythol) \[Easy\]](https://osu.ppy.sh/beatmaps/186601)(1.28 Yıldız)
+- [Akcent feat. Edward Maya - That's My Name (Barney Stinson) \[Easy\]](https://osu.ppy.sh/beatmaps/132838)(1.28 Yıldız)
+- [Hatsune Miku - Do-re-mi-fa Rondo (Sylphi) \[Easy\]](https://osu.ppy.sh/beatmaps/163457)(1.28 Yıldız)
+
+### Zorlu
+
+Kendinizi zorlayıp bir meydan okuma üstlenmek mi istiyorsunuz? Bunları deneyin!
+
+- [DJ Fresh - Gold Dust (galvenize) \[Easy\]](https://osu.ppy.sh/beatmaps/96042)(1.32 Yıldız)
+- [Kyary Pamyu Pamyu - Ninjya Re Bang Bang (IchiGUN) \[Easy\]](https://osu.ppy.sh/beatmaps/230174)(1.36 Yıldız)
+- [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix) (Bakari) \[Easy\]](https://osu.ppy.sh/beatmaps/172736)(1.36 Yıldız)
+- [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix) (Ami) \[Easy\]](https://osu.ppy.sh/beatmaps/368307)(1.45 Yıldız)
+- [Gentle Stick X M2U - Hades in the Heaven (AngelHoney) \[Easy\]](https://osu.ppy.sh/beatmaps/318994)(1.46 Yıldız)
+
+## Daha Ötesi
+
+Burada sıralanmış beatmapler size zor gelmemeye başladığında durmayın. Daha yüksek zorluk seviyelerindeki beatmapleri bulun ve devam edin. [Tavsiye edilen zorluğa göre](https://osu.ppy.sh/beatmapsets?c=recommended) beatmapleri aratmayı bile deneyebilirsiniz, zira bu bundan sonra nereye gitmeniz gerektiği hakkında sizlere bir fikir verecektir. Diğer oyunculara gelişmek hakkında osru sormaktan çekinmeyin, zira büyük ihtimalle onlar da iyi insanlar olup onların da osu! dünyası maceralarının başladığı, kendilerinin de eğlenerek oynadıkları bir beatmap paylaşacaklardır. Son olarak, ama en önemlisi, eğlencenize bakın!
```

Comment:
`Daha yüksek zorluk seviyelerindeki mapleri bulun ve devam edin.`
*it didn't show where exactly I suggested change*